plugins/aws/lex/lexAudioLogsEncrypted.js

```diff
@@ -0,0 +1,148 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Audio Logs Encrypted',
+    category: 'Lex',
+    domain: 'Content Delivery',
+    description: 'Ensure that Amazon Lex audio logs are encrypted',
```

Comment:
Ensure that Amazon Lex audio logs are encrypted using desired KMS encryption level.